clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -690,6 +690,80 @@ struct Table {
   }
 };
 
+// Add any llvm-spirv option that relies on a specific Triple in addition
```

Comment:
The same comment should be added into clang driver codebase. For now old offloading model is a default and therefore it is more likely than someone will update that part of the code without updating `clang-linker-wrapper`.

Reply:
Good catch. I will add the comment. We eventually would like to 'unify' the two functions. Hopefully soon. 

Thanks